tests/boot/boot_from_pxe.pm

```diff
@@ -196,7 +196,7 @@ sub run {
                 assert_screen $ssh_vnc_tag, $ssh_vnc_wait_time;
             }
         }
-        if (!is_upgrade && !get_var('KEEP_DISKS')) {
+        if (!is_upgrade && !get_var('KEEP_DISKS') && is_sle) {   #limite to SLE as those commands are not supported in openSUSE
```

Comment:
This is not very clear to me. Why we don't prepare disks in openSUSE? Which commands are not supported in openSUSE?

Reply:
openSUSE TW seems to use zRAM. prepare_disks() function fails to wipe some zram disks, please look at http://10.67.129.51/tests/2506#step/boot_from_pxe/39

I just skip it to save effort. Let me go further and see if there is any better way to solve this problem

Reply:
push a new commit to exclude /dev/zram when wiping existing disks.

Reply:
Thank you